crates/provider/src/builder.rs

```diff
@@ -99,17 +98,16 @@ impl<L, N> ProviderBuilder<L, N> {
     /// [`tower::ServiceBuilder`]. The first layer added will be the first to
```

Comment:
maybe add `pub fn network<Other: Network>(self) -> ProviderBuilder<Stack<Inner, L>, Other>` for lazy users or programmatic switching?

Reply:
Already exists https://github.com/alloy-rs/alloy/blob/9a33128261e4c1c48d7e124758758aa218024b20/crates/provider/src/builder.rs#L114-L125

Reply:
Just noticed docs need an update. cc @onbjerg

Reply:
yes Ethereum by default